tests/main/views/test_brief_response.py

```diff
@@ -729,6 +729,61 @@ def test_list_brief_responses_for_brief_id(self):
         assert all(br['briefId'] == another_brief_id for br in data['briefResponses'])
         assert 'self' in data['links']
 
+    def test_list_brief_responses_by_one_framework_slug(self, live_dos2_framework):
+        with self.app.app_context():
+            dos2_brief = Brief(
```

Comment:
I'd be tempted to keep it as it is.

The whole file relies on a brief created `BaseBriefResponseTest.setup` so the similar thing to do would be to add the DOS2 brief to that set up. But we only need the DOS2 brief for two of the tests in this file as opposed to all 53 of them so it doesn't feel appropriate to put it there.

Unless we were going to move the whole test set up to be done using fixtures, I'd be tempted not to mix them and have this file responsible for setting itself up.

Do you feel strongly either way?